docs/engineering-defaults.md

```diff
@@ -1,7 +1,147 @@
 # Engineering Defaults
 
+Much of Open O11y's guidance is based on or enhanced by these best practices for
+engineering teams.
+
 ## Pair Programming
 
-## Trunk Based Development
+Pair programming is a software development technique in which two developers work
+together at one workstation. One, the driver, writes code while the other, the
+observer or navigator, reviews each line of code as it is typed in. The two
+developers switch roles frequently.
+
+This method provides extremely fast feedback to developers about their code
+changes and design decisions before. In turn, this prevents wasted effort and produces
+cleaner, simpler code at a faster rate.
+
+Tooling can greatly improve the developers' experience while pair programming.
+Audio calls and screen sharing enable pair programming for engineers in
+different physical locations. Similarly, IDE tools such as Visual Studio Code's
+[Live Share][liveshare] extension allow both parties to develop code changes
+simultaneously in a single workspace.
 
 ## Small Batch Delivery
+
+Open O11y encourages teams to deliver in small batches. Planning small portions
+of work enables teams to deploy, demo, and request feedback more frequently.
+Much like pair programming, this practice aids teams in coordinating their
+efforts and safeguards against individuals investing time and energy in
+developing an incorrect solution.
+
+## Trunk Based Development
+
+> A source-control branching model, where developers collaborate on code in a
+> single branch called 'trunk'[^trunksummary]
+
+Open O11y highly recommends using [Trunk Based Development][trunk] over other
+branching models. Its focus on simplicity offers many advantages:
+
+* The 'main' branch represents the application and is always production-ready
+* Smaller, short-lived branches are dramatically safer to merge
+* Developers enjoy a reduced cognitive load
+* CI/CD requires less maintenance and/or is much simpler
+
+[Gitflow][gitflow]
```

Comment:
I seriously wonder if we should include the gitflowsucks link here.

Reply:
I considered it too. It has good info. And it would be funny

Reply:
@RobertKelly - any objections to us including gitflowsucks.com here?

Reply:
Up to you!